internal/service/sesv2/dedicated_ip_assignment_test.go

```diff
@@ -0,0 +1,154 @@
+package sesv2_test
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go-v2/service/sesv2/types"
+	sdkacctest "github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-aws/internal/acctest"
+	"github.com/hashicorp/terraform-provider-aws/internal/conns"
+	"github.com/hashicorp/terraform-provider-aws/internal/create"
+	tfsesv2 "github.com/hashicorp/terraform-provider-aws/internal/service/sesv2"
+	"github.com/hashicorp/terraform-provider-aws/names"
+)
+
+func TestAccSESV2DedicatedIPAssignment_serial(t *testing.T) {
+	testCases := map[string]func(t *testing.T){
+		"basic":      testAccSESV2DedicatedIPAssignment_basic,
+		"disappears": testAccSESV2DedicatedIPAssignment_disappears,
+	}
+
+	for name, tc := range testCases {
+		tc := tc
+		t.Run(name, func(t *testing.T) {
+			tc(t)
+		})
+	}
+}
+
+func testAccSESV2DedicatedIPAssignment_basic(t *testing.T) { // nosemgrep:ci.sesv2-in-func-name
+	if os.Getenv("SES_DEDICATED_IP") == "" {
```

Comment:
Document this environment variable in `docs/acc-test-environment-variables.md`.

Reply:
👍 thanks! done.